surefire-grouper/pom.xml

```diff
@@ -32,12 +32,16 @@
   <name>Maven Surefire Test-Grouping Support</name>
   <description>Maven Surefire Test-Grouping Support</description>
 
+  <properties>
+    <javacc.version>7.0.10</javacc.version>
```

Comment:
You can inline this value, that's not a problem for us, we will understand the point anyway.

Reply:
OK